### PR TITLE
update doc type

### PR DESCRIPTION
**Pull request by alonp99**
### Description
Elaborate on the subject, motivation, and context.

### Related issues
 * Provide a link to each related issue.

### Breaking changes
 * Describe the breaking changes that this pull request introduces.

### How these changes were tested
 * Describe the tests that you ran to verify your changes, including devices, operating systems, browsers and versions.

### Examples and references
 * Links, screenshots, and other resources related to this change.

### Checklist
- [] I have read the [contribution guidelines](CONTRIBUTING.md) of this project
- [] I have read the [style guidelines](STYLE_GUIDE.md) of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings and errors
- [] New and existing tests pass locally with my changes

### CODE REVIEW

File: services/workflows-service/src/workflow/update-documents.test.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { DefaultContextSchema } from '@/workflow/schemas/context';
+
 import { updateDocuments } from '@/workflow/update-documents';
+import {DefaultContextSchema} from "@ballerine/common";
```
Comment (alonp99) [inline services/workflows-service/src/workflow/update-documents.test.ts:3]: format

File: services/workflows-service/src/workflow/workflow.controller.internal.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ import { enrichWorkflowRuntimeData } from './enrich-workflow-runtime-data';
 import { UseGuards } from '@nestjs/common';
 import { WorkflowAssigneeGuard } from '@/auth/assignee-asigned-guard.service';
 import { WorkflowAssigneeId } from '@/workflow/dtos/workflow-assignee-id';
+import console from "console";
```
Comment (alonp99) [inline services/workflows-service/src/workflow/workflow.controller.internal.ts:27]: remove

File: apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx [hunk 3]
```diff
@@ -103,26 +111,47 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
...
-                        autoComplete={'off'}
-                        {...field}
-                      />
+                      {dropdownOptions ? (
+                        <Select
+                          disabled={!isEditable}
+                          onValueChange={field.onChange}
+                          defaultValue={field.value}
+                        >
+                          <SelectTrigger className="w-full">
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx:120]: `<FormControl>` is only supposed to wrap `<SelectTrigger>`.

File: apps/backoffice-v2/src/pages/Entity/hooks/useEntity/useEntity.tsx [hunk 4]
```diff
@@ -7,21 +7,74 @@ import { useCaseState } from '../../components/Case/hooks/useCaseState/useCaseSt
...
 import { components } from './components';
+import { getDocumentsByCountry } from '@ballerine/common/src/schemas/documents/workflow/documents/schemas/index';
 
 const convertSnakeCaseToTitleCase = (input: string): string =>
   input
     .split('_')
     .map(word => word.charAt(0).toUpperCase() + word.slice(1))
     .join(' ');
 
+const extractCountryCodeFromEntity = entity => {
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/hooks/useEntity/useEntity.tsx:18]: Should be `get` instead of `extract`.

**Review by Omri-Levy — APPROVED** [review]

File: apps/backoffice-v2/src/pages/Entity/hooks/useEntity/useEntity.tsx [hunk 5]
```diff
@@ -112,15 +171,28 @@ export const useEntity = () => {
...
                           title,
-                          value: properties?.[title] ?? '',
-                          type,
-                          format,
-                          pattern,
-                          isEditable: caseState.writeEnabled && isEditable,
-                        }),
+                          { type, format, pattern, isEditable = true, dropdownOptions, value },
+                        ]) => {
+                          const fieldValue = value || (properties?.[title] ?? '');
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/hooks/useEntity/useEntity.tsx:184]: Should either swap the positions of the condition or make a more explicit check to decide which value to use for example using `type`, `format`, or `dropdownOptions`.

**Pull request merged**